### DISCUSSION TITLE
Does linq2db support owned entities?

### DISCUSSION THREAD

**Discussion by cm4ker** [linq2db/linq2db#3725] [Q&A] [answered]
https://docs.microsoft.com/en-us/ef/core/modeling/owned-entities

On trying query entity witch has owned entity type I have an error like this: 

LinqToDB.Linq.LinqException: Association key 'MyTypeId' not found for type 'Namespace.MyType'.

Workaround: I create second model (only for linq2db) witch hasn't OET property, but for this I need to add so many code for prevent migration and I need always think abount this type then I make changes on base model

**Comment by MaceWindu** [comment, marked as answer]
There is support for nested classes mapping, which could partially replace owned entities. Check this example https://github.com/linq2db/linq2db/blob/master/Tests/Model/ComplexPerson.cs#L7-L9

**Reply by cm4ker** [reply]
I check this and it works. Thank you!
